graph/telemetry/istio/appender/appender.go

```diff
@@ -316,3 +340,40 @@ func getAppWorkloads(namespace, app, version string, gi *graph.AppenderGlobalInf
 	}
 	return result
 }
+
+func getApp(namespace, appName string, gi *graph.AppenderGlobalInfo) (*models.AppListItem, bool) {
```

Comment:
Any reason you didn't model this like the workloads with a `getApp` and `getAppList`?  Not a blocker.

Reply:
wasn't needed so opted not to add more API just to add it. I just added the API that was needed. If anyone needs a getAppList in the future, that can be implemented.